Building greenworks "unresolved external symbol"

After running npm install I get this:
```
minizip.lib(unzip.obj) : error LNK2001: unresolved external symbol inflateEnd [E:\Git\greenworks\build\greenworks-win64.vcxproj]
minizip.lib(unzip.obj) : error LNK2001: unresolved external symbol inflate [E:\Git\greenworks\build\greenworks-win64.vcxproj]
minizip.lib(unzip.obj) : error LNK2001: unresolved external symbol inflateInit2_ [E:\Git\greenworks\build\greenworks-win64.vcxproj]
minizip.lib(unzip.obj) : error LNK2001: unresolved external symbol crc32 [E:\Git\greenworks\build\greenworks-win64.vcxproj]
minizip.lib(zip.obj) : error LNK2001: unresolved external symbol deflateInit2_[E:\Git\greenworks\build\greenworks-win64.vcxproj]
minizip.lib(zip.obj) : error LNK2001: unresolved external symbol get_crc_table[E:\Git\greenworks\build\greenworks-win64.vcxproj]
minizip.lib(zip.obj) : error LNK2001: unresolved external symbol deflateEnd [E:\Git\greenworks\build\greenworks-win64.vcxproj]
minizip.lib(zip.obj) : error LNK2001: unresolved external symbol deflate [E:\Git\greenworks\build\greenworks-win64.vcxproj]
E:\Git\greenworks\build\Release\greenworks-win64.node : fatal error LNK1120: 8unresolved externals [E:\Git\greenworks\build\greenworks-win64.vcxproj]
```

I have Visual Studio 2015 installed, not sure if that could affect it.

--- a/deps/zlib/contrib/minizip/unzip.c
+++ b/deps/zlib/contrib/minizip/unzip.c
@@ -72,7 +72,7 @@
         #define NOUNCRYPT
 #endif
 
-#include "zlib.h"
+#include "../../zlib.h"
 #include "unzip.h"
 
 #ifdef STDC

--- a/deps/zlib/contrib/minizip/unzip.h
+++ b/deps/zlib/contrib/minizip/unzip.h
@@ -48,7 +48,7 @@ extern "C" {
 #endif
 
 #ifndef _ZLIB_H
-#include "zlib.h"
+#include "../../zlib.h"
 #endif
 
 #ifndef  _ZLIBIOAPI_H

--- a/deps/zlib/contrib/minizip/zip.c
+++ b/deps/zlib/contrib/minizip/zip.c
@@ -26,7 +26,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <time.h>
-#include "zlib.h"
+#include "../../zlib.h"
 #include "zip.h"
 
 #ifdef STDC

--- a/deps/zlib/contrib/minizip/zip.h
+++ b/deps/zlib/contrib/minizip/zip.h
@@ -47,7 +47,7 @@ extern "C" {
 //#define HAVE_BZIP2
 
 #ifndef _ZLIB_H
-#include "zlib.h"
+#include "../../zlib.h"
 #endif
 
 #ifndef _ZLIBIOAPI_H

